src/js/components/TextInput/TextInput.js

```diff
@@ -392,7 +392,10 @@ const TextInput = forwardRef(
                   else if (!theme.button.option)
                     // don't have theme support, need to layout here
                     child = (
-                      <Box align="start" pad="small">
+                      <Box
+                        align="start"
+                        pad={theme.textInput.suggestions?.container?.pad}
```

Comment:
It looks like we only use this theme prop when `theme.button.option` isn't defined.

The name `theme.textInput.suggestions?.container?.pad` makes me think that this would be applied to all suggestions in the TextInput. So maybe we want a more specific name to indicate that this is only applied sometimes?

Not sure what name we want to use here, this is a pretty narrow case when we use this padding